modules/Contacts/JAccountMissingException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+
+class JAccountMissingException extends Exception {}
```

Comment:
As described earlier, please consider using the construct to set the default message and error code, so that it will provide better error messages for logging.